scripts/scheduler.py

```diff
@@ -357,11 +357,12 @@ def queue_new_jobs(available_slots, env, config, studies, wes_token, exclude_stu
         # support resume, detect whether run info file exists, if so get session id
         resume = False  # set resume to the session id, set to None for now
 
-        exist_run_path = os.path.join(job, f'run.*.{env}.wes-*')
+        exist_run_path = os.path.join(job, f'run.*.*.wes-*')
         exist_runs = sorted(glob(exist_run_path))
         if exist_runs:
           run_file = os.path.basename(exist_runs[-1])
-          latest_run_id = run_file.split('.')[-1]
+          latest_env, latest_run_id = run_file.split('.')[-2:]
+          if not latest_env == env: continue
```

Comment:
so the logic is to find all previous runs regardless of env, then find the last run. If the last run was not the current env, skip it. If it is, then retry to resume it. That's good